packages/land/contracts/Land.sol

```diff
@@ -88,10 +94,15 @@ contract Land is LandBaseToken, OperatorFiltererUpgradeable {
     }
 
     /// @notice set royalty manager
-    /// @param _royaltyManager address of royalty manager to set
-    function _setRoyaltyManager(address _royaltyManager) internal {
-        royaltyManager = IRoyaltyManager(_royaltyManager);
-        emit RoyaltyManagerSet(_royaltyManager);
+    /// @param royaltyManager address of royalty manager to set
+    function _setRoyaltyManager(address royaltyManager) internal {
+        _royaltyManager = IRoyaltyManager(royaltyManager);
+        emit RoyaltyManagerSet(royaltyManager);
+    }
+
+    function _transferOwnership(address newOwner) internal {
+        _owner = newOwner;
+        emit OwnershipTransferred(_owner, newOwner);
```

Comment:
Sorry, it's wrong now because `_owner` contains the new owner.
Revert it back to what you did i guess
and add a test to confirm this